apps/ui/config/dev.exs

```diff
@@ -30,13 +30,13 @@ config :fortune, FortuneWeb.Endpoint,
 # different ports.
 
 # Watch static and templates for browser reloading.
-config :fortune, FortuneWeb.Endpoint,
+config :ui, UiWeb.Endpoint,
   live_reload: [
     patterns: [
       ~r{priv/static/.*(js|css|png|jpeg|jpg|gif|svg)$},
       ~r{priv/gettext/.*(po)$},
-      ~r{lib/fortune_web/views/.*(ex)$},
-      ~r{lib/fortune_web/templates/.*(eex)$}
+      ~r{lib/ui_web/views/.*(ex)$},
```

Comment:
let's flatten out the structure. By default, Phoenix generates the paths that are nested in a way I fucking hate.

So we have lib/ui_web/templates etc. the ui_web is totally unnecessary. Let's remove this directory and put everything straight to lib/. So that we have:

lib/views
lib/models
lib/channels
lib/templates

etc etc.